docs/fsharp/language-reference/compiler-options.md

```diff
@@ -15,53 +15,53 @@ The following table shows compiler options listed alphabetically. Some of the F#
+|`--define:symbol`<br /><br />`-d:symbol`|Defines a symbol for use in conditional compilation.|
+|<code>--deterministic[+&#124;-]</code>|Produce a deterministic assembly (including module version GUID and timestamp).  This cannot be used with wildcard version numbers, and only supports embedded and portable debugging types|
+|`--doc:xmldoc-filename`|Instructs the compiler to generate XML documentation comments to the file specified. For more information, see [XML Documentation](xml-documentation.md).<br /><br />This compiler option is equivalent to the C# compiler option of the same name. For more information, see [&#47;doc &#40;C&#35; Compiler Options&#41;](https://msdn.microsoft.com/library/3260k4x7.aspx).|
+|`--fullpaths`|Instructs the compiler to generate fully qualified paths.<br /><br />This compiler option is equivalent to the C# compiler option of the same name. For more information, see [&#47;fullpaths &#40;C&#35; Compiler Options&#41;](https://msdn.microsoft.com/library/d315xc66.aspx).|
+|`--help`<br /><br />`-?`|Displays usage information, including a brief description of all the compiler options.|
+|<code>--highentropyva[+&#124;-]</code>|Enable or disable high-entropy address space layout randomization (ASLR), an enhanced security feature. The OS randomizes the locations in memory where infrastructure for applications (such as the stack and heap) are loaded. If you enable this option, operating systems can use this randomization to use the full 64-bit address-space on a 64-bit machine.|
+|`--keycontainer:key-container-name`|Specifies a strong name key container.|
+|`--keyfile:filename`|Specifies the name of a public key file for signing the generated assembly.|
+|`--lib:folder-name`<br /><br />`-I:folder-name`|Specifies a directory to be searched for assemblies that are referenced.<br /><br />This compiler option is equivalent to the C# compiler option of the same name. For more information, see [&#47;lib &#40;C&#35; Compiler Options&#41;](https://msdn.microsoft.com/library/s5bac5fx.aspx).|
+|`--linkresource:resource-info`|Links a specified resource to the assembly. The format of resource-info is <code>filename[name[public&#124;private]]</code><br /><br />Linking a single resource with this option is an alternative to embedding an entire resource file with the `--resource` option.<br /><br />This compiler option is equivalent to the C# compiler option of the same name. For more information, see [&#47;linkresource &#40;C&#35; Compiler Options&#41;](https://msdn.microsoft.com/library/xawyf94k.aspx).|
+|`--mlcompatibility`|Ignores warnings that appear when you use features that are designed for compatibility with other versions of ML.|
```

Comment:
define what the ML acronym means?

Reply:
In context, ML is the correct term (since this flag is for compatibility with other ML dialects). ML is short for Hindley **M**i**L**ner (emphasis mine), a family of languages that F# is a part of.